srcDoc/release_notes.rst

```diff
@@ -48,6 +48,19 @@ releases:
 Release Notes Master
 --------------------
 
+What's New in Master
+~~~~~~~~~~~~~~~~~~~~
+
+Ability to use your own type in RTI Perftest
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+*RTI Perftest* now supports the ability to use your own type in RTI Perftest.
```

Comment:
... ability to use your own custom type.
(no need to say "RTI Perftest" again at the end)